Assets/HoloToolkit/Utilities/Scripts/Layers.cs

```diff
@@ -0,0 +1,67 @@
+﻿//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+	public static class Layers
```

Comment:
I'm pretty sure we might already have a class that does layers, or layer extensions. Should this logic be there?

Reply:
I wanted to fold this into LayerExtensions but was talked out of it as this isn't technically an extension. I'd be happy to consolidate.